### PR TITLE
Note on double encoding of Storage Key

### PR DESCRIPTION
**Pull request by WillEastbury**
in K8S Secrets..

Added *If you place the storageAccessKey parameter (under the spec/metadata path) into the secret store, it needs to be Base64 encoded AGAIN in the yaml manifest file, (even though it is already in a base64 encoded format).*

Thank you for helping make the Dapr documentation better!

If you are a new contributor, please see the [this contribution guidance](https://github.com/dapr/docs/blob/master/contributing/README.md) which helps keep the Dapr documentation readable, consistent and useful for Dapr users.

If you are contributing a new authored "How To" article, [this template](https://github.com/dapr/docs/blob/master/contributing/howto-template.md) was created to assist you.

In addition, please fill out the following to help reviewers understand this pull request:

## Description

_Please explain the changes you've made_

## Issue reference

_Please reference the issue this PR will close: #[issue number]_

### CODE REVIEW

File: reference/specs/bindings/storagequeues.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ spec:
 - `queue` is the name of the Azure Storage queue.
 - `ttlInSeconds` is an optional parameter to set the default message time to live. If this parameter is omitted, messages will expire after 10 minutes.
 
-> **Note:** In production never place passwords or secrets within Dapr components. For information on securely storing and retrieving secrets refer to [Setup Secret Store](../../../howto/setup-secret-store)
+> **Note:** In production never place passwords or secrets within Dapr components. For information on securely storing and retrieving secrets refer to [Setup Secret Store](../../../howto/setup-secret-store) *If you place the storageAccessKey parameter (under the spec/metadata path) into the secret store, it needs to be Base64 encoded AGAIN in the yaml manifest file, (even though it is already in a base64 encoded format).*
```
Comment (youngbupark) [inline reference/specs/bindings/storagequeues.md:27]: @yaron2 can you please double-check this? do we need double-encoding for accesskey? I haven't known it.

Comment (yaron2) [reply reference/specs/bindings/storagequeues.md:27]: Yes. If this is true we need to have this comment on every component that is based on Azure Storage.

Comment (yaron2) [reply reference/specs/bindings/storagequeues.md:27]: I tested this and can confirm this is not the case.

I created a Kubernetes secret with the raw storage key from Azure (which is base64 encoded) and did not need to base64 encode it again.

**Comment by yaron2** [comment]
Thanks for opening the issue @WillEastbury.

I [tried reproducing](https://github.com/dapr/docs/pull/676#discussion_r496041541) this and couldn't.

I am closing for now, feel free to reopen if you can provide exact reproduction steps.

**Pull request closed without merging**